src/stdlib_experimental_io.f90

```diff
@@ -265,17 +266,7 @@ integer function number_of_rows_numeric(s)
 
 end function
 
-pure logical function whitechar(char) ! white character
-! returns .true. if char is space (32) or tab (9), .false. otherwise
-character, intent(in) :: char
-if (iachar(char) == 32 .or. iachar(char) == 9) then
-    whitechar = .true.
-else
-    whitechar = .false.
-end if
-end function
-
-integer function open(filename, mode) result(u)
+integer function open(filename, mode, io) result(u)
```

Comment:
Should the `io` be called `iostat`?

Reply:
I usually don't like to use official names. However, in this case using `iostat` makes sense.